app/renderer/components/common/messageBox.js

```diff
@@ -3,20 +3,34 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const React = require('react')
-const ImmutableComponent = require('../immutableComponent')
+const {StyleSheet, css} = require('aphrodite')
```

Comment:
`cx()` is just a lib to apply styles conditionally, so it's affected by the same rules as CSS like deeply nested selectors taking precedence, and `!important` which overrides everything. 

I don't think there's anything we can do with that but just avoid adding new LESS stuff and keep using until it can be removed.